{"uid":"2b8e6d69603f694","name":"test_has_new_assets[Test for DAO IPCI]","fullName":"tests.test_check_new_assets.TestAssets#test_has_new_assets","historyId":"dc86880bf1eea353867bbdf7bd9268e8","time":{"start":1726391157931,"stop":1726391157931,"duration":0},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","statusTrace":"timeout = 15\nfunction = <bound method Chain.create_connection of <scripts.utils.chain_model.Chain object at 0x7fdcaed5b790>>\nargs = (), error_message = 'Timeout 15 when creating connection: DAO IPCI'\n\n    def execute_with_timeout(timeout: int, function, args: tuple, error_message: str):\n        try:\n>           return func_timeout(timeout=timeout, func=function, args=args)\n\ntests/conftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ntimeout = 15\nfunc = <bound method Chain.create_connection of <scripts.utils.chain_model.Chain object at 0x7fdcaed5b790>>\nargs = (), kwargs = {}\n\n    def func_timeout(timeout, func, args=(), kwargs=None):\n        '''\n            func_timeout - Runs the given function for up to #timeout# seconds.\n    \n            Raises any exceptions #func# would raise, returns what #func# would return (unless timeout is exceeded), in which case it raises FunctionTimedOut\n    \n            @param timeout <float> - Maximum number of seconds to run #func# before terminating\n    \n            @param func <function> - The function to call\n    \n            @param args    <tuple> - Any ordered arguments to pass to the function\n    \n            @param kwargs  <dict/None> - Keyword arguments to pass to the function.\n    \n    \n            @raises - FunctionTimedOut if #timeout# is exceeded, otherwise anything #func# could raise will be raised\n    \n            If the timeout is exceeded, FunctionTimedOut will be raised within the context of the called function every two seconds until it terminates,\n            but will not block the calling thread (a new thread will be created to perform the join). If possible, you should try/except FunctionTimedOut\n            to return cleanly, but in most cases it will 'just work'.\n    \n            @return - The return value that #func# gives\n        '''\n    \n        if not kwargs:\n            kwargs = {}\n        if not args:\n            args = ()\n    \n        ret = []\n        exception = []\n        isStopped = False\n    \n        def funcwrap(args2, kwargs2):\n            try:\n                ret.append( func(*args2, **kwargs2) )\n            except FunctionTimedOut:\n                # Don't print traceback to stderr if we time out\n                pass\n            except Exception as e:\n                exc_info = sys.exc_info()\n                if isStopped is False:\n                    # Assemble the alternate traceback, excluding this function\n                    #  from the trace (by going to next frame)\n                    # Pytohn3 reads native from __traceback__,\n                    # python2 has a different form for \"raise\"\n                    e.__traceback__ = exc_info[2].tb_next\n                    exception.append( e )\n    \n        thread = StoppableThread(target=funcwrap, args=(args, kwargs))\n        thread.daemon = True\n    \n        thread.start()\n        thread.join(timeout)\n    \n        stopException = None\n        if thread.is_alive():\n            isStopped = True\n    \n            class FunctionTimedOutTempType(FunctionTimedOut):\n                def __init__(self):\n                    return FunctionTimedOut.__init__(self, '', timeout, func, args, kwargs)\n    \n            FunctionTimedOutTemp = type('FunctionTimedOut' + str( hash( \"%d_%d_%d_%d\" %(id(timeout), id(func), id(args), id(kwargs))) ), FunctionTimedOutTempType.__bases__, dict(FunctionTimedOutTempType.__dict__))\n    \n            stopException = FunctionTimedOutTemp\n            thread._stopThread(stopException)\n            thread.join(min(.1, timeout / 50.0))\n>           raise FunctionTimedOut('', timeout, func, args, kwargs)\nE           func_timeout.exceptions.FunctionTimedOut: Function create_connection (args=()) (kwargs={}) timed out after 15.000000 seconds.\n\n.venv/lib/python3.10/site-packages/func_timeout/dafunc.py:101: FunctionTimedOut\n\nDuring handling of the above exception, another exception occurred:\n\nrequest = <SubRequest 'chain_model' for <Function test_has_new_assets[Test for DAO IPCI]>>\n\n    @pytest.fixture(scope=\"module\", params=get_substrate_chains(), ids=chain_names)\n    def chain_model(request) -> Chain:\n        chain = request.param\n>       execute_with_timeout(\n            timeout=FIXTURE_TIMEOUT,\n            function=chain.create_connection,\n            args=(),\n            error_message=f'Timeout {FIXTURE_TIMEOUT} when creating connection: {chain.name}'\n        )\n\ntests/conftest.py:58: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ntimeout = 15\nfunction = <bound method Chain.create_connection of <scripts.utils.chain_model.Chain object at 0x7fdcaed5b790>>\nargs = (), error_message = 'Timeout 15 when creating connection: DAO IPCI'\n\n    def execute_with_timeout(timeout: int, function, args: tuple, error_message: str):\n        try:\n            return func_timeout(timeout=timeout, func=function, args=args)\n        except FunctionTimedOut:\n            with allure.step(\"Function timed out\"):\n                allure.attach(name=\"Error\", body=error_message,\n                              attachment_type=allure.attachment_type.TEXT)\n>           raise FixtureTimeoutError(error_message)\nE           tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI\n\ntests/conftest.py:34: FixtureTimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":2,"retriesStatusChange":false,"beforeStages":[{"name":"chain_model","time":{"start":1726391117677,"stop":1726391132787,"duration":15110},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 58, in chain_model\n    execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 34, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1726391132787,"stop":1726391132787,"duration":0},"status":"passed","steps":[],"attachments":[{"uid":"545dcc57c3fa5618","name":"Error","source":"545dcc57c3fa5618.txt","type":"text/plain","size":45}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},{"name":"chain_model","time":{"start":1726391137804,"stop":1726391152915,"duration":15111},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 58, in chain_model\n    execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 34, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1726391152914,"stop":1726391152914,"duration":0},"status":"passed","steps":[],"attachments":[{"uid":"19c0cfed3e4e1621","name":"Error","source":"19c0cfed3e4e1621.txt","type":"text/plain","size":45}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},{"name":"chain_model","time":{"start":1726391157931,"stop":1726391173041,"duration":15110},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 58, in chain_model\n    execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 34, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1726391173041,"stop":1726391173041,"duration":0},"status":"passed","steps":[],"attachments":[{"uid":"c772492d3c4e7a68","name":"Error","source":"c772492d3c4e7a68.txt","type":"text/plain","size":45}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_check_new_assets"},{"name":"subSuite","value":"TestAssets"},{"name":"host","value":"fv-az1205-438"},{"name":"thread","value":"2119-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_check_new_assets"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain_model","value":"<scripts.utils.chain_model.Chain object at 0x7fdcaed5b790>"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"e08e5be142a45ef5","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726391137804,"stop":1726391137804,"duration":0}},{"uid":"94635d602ab38b42","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726391117675,"stop":1726391117675,"duration":0}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":45,"skipped":0,"passed":94,"unknown":0,"total":139},"items":[{"uid":"7c0daf09efef6005","reportUrl":"https://novasamatech.github.io/test-runner/5708//#testresult/7c0daf09efef6005","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726326341412,"stop":1726326341412,"duration":0}},{"uid":"520dc87c19956558","reportUrl":"https://novasamatech.github.io/test-runner/5707//#testresult/520dc87c19956558","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726304744417,"stop":1726304744417,"duration":0}},{"uid":"c7dcf24cba7efc1c","reportUrl":"https://novasamatech.github.io/test-runner/5706//#testresult/c7dcf24cba7efc1c","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726239970032,"stop":1726239970032,"duration":0}},{"uid":"469572d4193adc5d","reportUrl":"https://novasamatech.github.io/test-runner/5705//#testresult/469572d4193adc5d","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726218352769,"stop":1726218352769,"duration":0}},{"uid":"ac0a7ddfe4f3b562","reportUrl":"https://novasamatech.github.io/test-runner/5704//#testresult/ac0a7ddfe4f3b562","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726153568486,"stop":1726153568486,"duration":0}},{"uid":"b9795ba65cabbfc4","reportUrl":"https://novasamatech.github.io/test-runner/5703//#testresult/b9795ba65cabbfc4","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726131959209,"stop":1726131959209,"duration":0}},{"uid":"c790fbede8c6794d","reportUrl":"https://novasamatech.github.io/test-runner/5702//#testresult/c790fbede8c6794d","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1726117791364,"stop":1726117791364,"duration":0}},{"uid":"8dbe351f4b22e5fc","reportUrl":"https://novasamatech.github.io/test-runner/5695//#testresult/8dbe351f4b22e5fc","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725808018344,"stop":1725808018344,"duration":0}},{"uid":"4aea4e9b33bf0efc","reportUrl":"https://novasamatech.github.io/test-runner/5694//#testresult/4aea4e9b33bf0efc","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725786348960,"stop":1725786348960,"duration":0}},{"uid":"9b122a9ad250611e","reportUrl":"https://novasamatech.github.io/test-runner/5693//#testresult/9b122a9ad250611e","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725721560275,"stop":1725721560275,"duration":0}},{"uid":"e06cf6f89484424d","reportUrl":"https://novasamatech.github.io/test-runner/5692//#testresult/e06cf6f89484424d","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725699959687,"stop":1725699959687,"duration":0}},{"uid":"d952819cde6f4b4d","reportUrl":"https://novasamatech.github.io/test-runner/5691//#testresult/d952819cde6f4b4d","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725635162135,"stop":1725635162135,"duration":0}},{"uid":"1690efe7c37b32b2","reportUrl":"https://novasamatech.github.io/test-runner/5690//#testresult/1690efe7c37b32b2","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725613565426,"stop":1725613565426,"duration":0}},{"uid":"c0f72069d3b60e06","reportUrl":"https://novasamatech.github.io/test-runner/5689//#testresult/c0f72069d3b60e06","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725548740425,"stop":1725548740425,"duration":0}},{"uid":"c117e4f6d6ff394e","reportUrl":"https://novasamatech.github.io/test-runner/5688//#testresult/c117e4f6d6ff394e","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725527158269,"stop":1725527158269,"duration":0}},{"uid":"e91f32db275a7288","reportUrl":"https://novasamatech.github.io/test-runner/5687//#testresult/e91f32db275a7288","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725462358707,"stop":1725462358707,"duration":0}},{"uid":"3042b3cbde5ada25","reportUrl":"https://novasamatech.github.io/test-runner/5686//#testresult/3042b3cbde5ada25","status":"broken","statusDetails":"tests.conftest.FixtureTimeoutError: Timeout 15 when creating connection: DAO IPCI","time":{"start":1725440758356,"stop":1725440758356,"duration":0}},{"uid":"cb872576fefd6cab","reportUrl":"https://novasamatech.github.io/test-runner/5685//#testresult/cb872576fefd6cab","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'get_constant'","time":{"start":1725375906521,"stop":1725375906521,"duration":0}},{"uid":"5a3ff3ff90ed2d03","reportUrl":"https://novasamatech.github.io/test-runner/5684//#testresult/5a3ff3ff90ed2d03","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'get_constant'","time":{"start":1725370333954,"stop":1725370333954,"duration":0}},{"uid":"b475a3a2458300cf","reportUrl":"https://novasamatech.github.io/test-runner/5684//#testresult/b475a3a2458300cf","status":"broken","statusDetails":"AttributeError: 'NoneType' object has no attribute 'get_constant'","time":{"start":1725368780813,"stop":1725368780813,"duration":0}}]},"tags":[]},"source":"2b8e6d69603f694.json","parameterValues":["<scripts.utils.chain_model.Chain object at 0x7fdcaed5b790>"]}